rpmconf/Install/Util/modules/packages.sh

```diff
@@ -296,7 +296,7 @@ findLatestPackage() {
             file_location="repo"
          fi
       else
-         if grep -q -w -e "^$package" <(yum list available -q -e 0 "$package" 2>/dev/null)
+         if grep -q -w -e "^$package" <(yum --show-duplicate list available -q -e 0 "$package" 2>/dev/null)
```

Comment:
Just querying this.  Would be good to know why you think this should fix the issue, especially as to me, `--show-duplicate` sounds suspiciously close but different to `--showduplicates` used just a few lines above.  Also http://www.unix.com/man-page/centos/8/yum/ makes no mention of `--show-duplicate` - but it does talk about `--showduplicates`.
Admittedly, RHEL 7.2 does appear to accept both forms.